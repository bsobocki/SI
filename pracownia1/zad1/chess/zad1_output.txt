9
6
7
6
8
10
