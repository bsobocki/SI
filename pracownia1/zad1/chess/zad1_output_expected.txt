out: 9
out: 6
out: 9
out: 6
out: 8
out: 10